apps/condo/domains/meter/access/MeterReading.js

```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
         return await checkPermissionsInEmployedOrRelatedOrganizations(context, user, meterOrganization, 'canManageMeterReadings')
     }
 
+    if (operation === 'update' && itemId) {
```

Comment:
It looks strange if you change the access logic and don’t create or update any test

Reply:
added tests for update logic